### PR TITLE
Implement ci-orchestrator in .github

### PR DESCRIPTION
**Pull request by kerbou**
Ref: https://app.zenhub.com/workspaces/the-outlaws-6193fe815d79fc0011e741b1/issues/gh/energinet-datahub/the-outlaws/953

### CODE REVIEW

File: .github/workflows/ci-orchestrator.yml [hunk 1]
```diff
@@ -6,10 +6,53 @@ on:
       - main
 
 jobs:
+  #
+  # License and Markdown Check
+  #
+  ci_base:
+    uses: Energinet-DataHub/.github/.github/workflows/ci-base.yml@v10
```
Comment (dstenroejl) [inline .github/workflows/ci-orchestrator.yml:13]: Should we refer to a version if we are in the repository?

Previously we used it "directly" by:
`uses: ./.github/workflows/ci-base.yml`

File: .github/workflows/ci-orchestrator.yml [hunk 2]
```diff
@@ -6,10 +6,53 @@ on:
...
+    with:
+      SKIP_LICENSE_CHECK: true
```
Comment (dstenroejl) [inline .github/workflows/ci-orchestrator.yml:15]: The repository is public, so should files not be forced to contain a license header?
If not, we might have to update our documentation to reflect that (I think I wrote somewhere that all public repositories should enable license check).

**Review by dstenroejl — COMMENTED** [review]
A few observations

File: .github/workflows/ci-orchestrator.yml (hunk 1 above)
Comment (kerbou) [reply .github/workflows/ci-orchestrator.yml:13]: You are absolutely correct, will fix this

File: .github/workflows/ci-orchestrator.yml (hunk 2 above)
Comment (kerbou) [reply .github/workflows/ci-orchestrator.yml:15]: Again, spot on, will fix this

**Review by dstenroejl — DISMISSED** [review]

File: .github/workflows/ci-orchestrator.yml [hunk 3]
```diff
@@ -6,10 +6,52 @@ on:
...
+      env:
+        GH_TOKEN: ${{ github.token }}
+
   allow_merge_ci_orchestrator:
     runs-on: ubuntu-latest
     needs: [
-
+      ci_base,
+      powershell_verification,
+      createrelease
```
Comment (Renetnielsen) [inline .github/workflows/ci-orchestrator.yml:54]: Change here also

**Pull request merged**